### ISSUE TITLE
example : jubaclassifier process aborted in parameter searcher

### ISSUE THREAD

**Issue by shiodat** [jubatus/jubakit#30]
The example `classifier_parameter.py` failed to finish while classifying dataset.
This problem occurs when using digits dataset, not artificial dataset.
https://gist.github.com/shiodat/2f6b85a0c2d4cfefa93d549dd2edc8a3

I found jubaclassifier process aborted while classifying dataset.
Any ideas?

**Comment by kmaehashi** [comment]
This is because the server process prints huge logs which exceeds the internal buffer.
In CW / regularization_weight = 10.0 case, the server tries to print the following log many times.

```
2016-07-21 13:54:06,657 23871 WARN  [classifier_serv.cpp:167] score is infinite: 3 = nan
```

There are two possible solutions:

1. Read log buffer appropriately (call `select` to the stdout file descriptor and read it from another thread)
2.  Redirect logs to on disk buffer.

The idea 1 is not cost-effective; in fact we don't need logs unless error cases (e.g., invalid config is given).  So I take the idea 2 in #31.

**Comment by shiodat** [comment]
This problem is solved via #31 .
